### DISCUSSION TITLE
2020-07-14-neil-seans-hall-of-fame/

### DISCUSSION THREAD

**Discussion by giscus** [FanzOfLenaZavaroni/fanzoflenazavaroni.github.io#21] [General]
# 2020-07-14-neil-seans-hall-of-fame/

The Sky television programme Jameson Tonight was produced in the studio. The show was very cheaply made and basically was trying to be like Johnny Carson in the US without the great host or indeed guests. The most notable I remembered as a guest was the wonderful and gifted Lena Zavaroni plus a great actor who will always been known as the TV favourite ‘Jason King’ aka Peter Wyngarde.

https://fanzoflenazavaroni.github.io/2020-07-14-neil-seans-hall-of-fame/

**Comment by StephenMolloy1** [comment]
A few days after her death, Derek mentioned that he had interviewed her on television but didn't give any clue as to when.

https://www.theargus.co.uk/news/6798928.derek-jameson-lena-zavaroni-and-the-fame-game/

**Comment by FanzOfLenaZavaroni** [comment]
Thanks for the link, sadly very little details are available for "Jameson Tonight", I checked the listings page for the programme via [The Times Digital Archive](https://galeapps.gale.com/apps/auth?userGroupName=bro_ttda&origURL=https%3A%2F%2Fgo.gale.com%2Fps%2FdispBasicSearch.do%3FuserGroupName%3Dbro_ttda%26prodId%3DTTDA&prodId=TTDA) but the "Times" never listed the guests on the programme so no joy there as to which of the dates was Lena's apperance on the programme and the [BFI](https://www2.bfi.org.uk/films-tv-people/5ebb4117a5d0b) page for "Jameson Tonight" is pointless as it has no details on the programme. There are a couple of full episodes on YouTube and a number of clips but none related to Lena. The IMDb only has the [1990](https://www.imdb.com/title/tt1830037/) addition of the programme.
